bread/bread.py

```diff
@@ -69,8 +69,8 @@ def get_full_perm_name(self, short_name):
 
     def __init__(self, *args, **kwargs):
         # Make sure the permission needed to use this view exists.
-        super(BreadViewMixin, self).__init__(*args, **kwargs)
-        perm_name = "%s_%s" % (
+        super().__init__(*args, **kwargs)
+        perm_name = "{}_{}".format(
```

Comment:
I noticed this was not updated to use f-strings. Is this done by pyupgrade?

Reply:
Yes, that was done by pyupgrade